### ISSUE TITLE
Audio recording capabilities

### ISSUE THREAD

**Issue by kryvosheiaivan** [IoTBits/ESP32_SGTL5000_driver#2]
Hello!
Tell me please, can audio recording be implemented on AudioBit board?
I mean, is it possible to connect microphone to the board, configure all that needed and then
save the audio file somewhere. After all, the audio file I would like to transfer through the Internet to some server.

**Comment by IoTBlocks** [comment]
Yes, the board has built-in microphone bias for condenser microphone with analog and digital volume control. You can then read the mic data via I2S at any bit depth/sampling rate you want. You can even mix that up with LINE IN or audio out. The codec has built-in DSP for all this so that the MCU does not have to do it.
You can finally just store/webstream the resulting final I2S data coming from the AudioBit.

**Comment by magnocube** [comment]
Hello!,
No issue, but i had a question.
in your implementation of: audiobit_write_reg (.....);  you used an array of 4 bits to store all the data (uint8_t dwr[4];).
You also commented out 4 lines with i2c_master_write_byte(.....); 
why did you choose to use the array instead of the i2c_master_write_byte? has this something to do with the working of i2c?, or is it only to be a bit more clear to read?

Thank you in advance!
